src/Nest/Indices/Monitoring/IndicesStats/ShardCommit.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	[JsonObject]
+	public class ShardCommit
+	{
+		[JsonProperty("id")]
+		public string Id { get; internal set; }
+
+		[JsonProperty("generation")]
+		public int Generation { get; internal set; }
+
+		[JsonProperty("user_data")]
+		public IDictionary<string, string> UserData { get; internal set; }
```

Comment:
should be `IReadOnlyDictionary<string, string>` and initialized to empty

Reply:
:+1: